server/svix-server/src/v1/utils/mod.rs

```diff
@@ -563,6 +564,88 @@ pub struct EventTypeNamePath {
     pub event_type_name: EventTypeName,
 }
 
+/// JsonStatus is a wrapper over `axum::extract::Json` as a handler output.
+/// Setting the `STATUS` const parameter automatically sets the response
+/// status code, as well as inserting it into the aide documentation.
+pub struct JsonStatus<const STATUS: u16, T: JsonSchema + Serialize>(pub T);
```

Comment:
I wonder if it's worth just calling this `Json` instead of `JsonStatus`. A bit overloady of `axum::Json`, but if we'll never use `axum::Json` after this maybe it's worth it.

Reply:
`axum::Json` will be used, because that's the extractor for JSON bodies, so the request side of things. Alternatively, we could choose to name this `Json` and always alias that to `JsonBody` for example, but I think the way it is at the moment is clearer.